common/pipes/pubspec.yaml

```diff
@@ -1,10 +1,11 @@
 name: pipes
-description: Angular Dart API Examples for Pipes
+description: AngularDart API Examples for Pipes
 version: 0.0.1
 environment:
   sdk: '>=1.19.0 <2.0.0'
 dependencies:
-  angular2: 2.0.0-beta.22
```

Comment:
Not yet, anyway. AngularDart is allowing itself to have breaking API changes in minor (x.*y*.z) releases.

Until this is fixed, we have to do the version constraining this way.